tests/unit/client_test.py

```diff
@@ -1,15 +1,143 @@
 import os
+
+import sure
```

Comment:
I think the main advantage is readability, which is a big benefit for unit
test code, which can get really obscured really fast. I definitely would
NOT use something like "sure" in production code.

---------
Matt Taylor
OS Community Flag-Bearer
Numenta

On Fri, May 8, 2015 at 9:44 AM, BoltzmannBrain <notifications@github.com>
wrote:

> In tests/unit/client_test.py
> <https://github.com/numenta/cortipy/pull/12#discussion_r29955258>:
>
> > @@ -1,15 +1,143 @@
> >  import os
> > +
> > +import sure
>
> I wouldn't say I hate them, but they're certainly new to me and seem
> unconventional. Kind of cool actually. Besides readability are there
> advantages over the unittest assert methods
> <https://docs.python.org/2/library/unittest.html#assert-methods>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numenta/cortipy/pull/12/files#r29955258>.
>